status.js

```diff
@@ -217,10 +215,23 @@ init()
       hydraLink.href = record['hydra_url'];
       hydraLink.innerText = [record['project'], record['jobset'], record['job']].join('/');
 
+      statusToColor = status => status ? "#b5ffb5" : "#ff9e9e";
+
       var hydra = row.getElementsByClassName("hydra")[0];
-      hydra.style.backgroundImage = "linear-gradient(to right, " +
-        (record['job_history'].map(val => val ? "#b5ffb5" : "#ff9e9e")).join(", ") +
-        ")";
+      switch (record['job_history'].length) {
+        case 0:
+          // Unknown, no color, leave it gray.
+          break;
+        case 1:
+          hydra.style.backgroundColor = statusToColor(record['job_history'][0]);
+          break;
+        default:
+          hydra.style.backgroundImage = "linear-gradient(to right, " +
+            (record['job_history'].map(statusToColor)).join(", ") +
+            ")";
+          break;
+      }
```

Comment:
Blah, the background rules for CSS are so inconvenient... It would have been much nicer if `linear-gradient(to right, #f00)` made it a flat red background.